src/cumin/core.clj

```diff
@@ -1,5 +1,5 @@
 (ns cumin.core
-  (:require [korma.core :refer :all]
+  (:require [korma.core :refer [order post-query select select* where]]
```

Comment:
The tests are failing. You need to also `:refer` to `defentity` -- see [here](https://github.com/esanmiguelc/cumin/blob/37316a8720e83114cb2dd734b65cae4d27bc73fd/src/cumin/core.clj#L54).